### ISSUE TITLE
points fail to pickle

### ISSUE THREAD

**Issue by saraedum** [sagemath/pplpy#30]
With pplpy 0.8.9 (from conda-forge) the following fails in SageMath 9.7:

```py
sage: loads(dumps(Polyhedron([(0, 0, 0)])))
AttributeError: type object 'type' has no attribute 'point'
```

On a lower level, this is:

```py
sage: from ppl import Generator, Variable
sage: loads(dumps(Generator.point(Variable(0)+Variable(1)+Variable(2))))
AttributeError: type object 'type' has no attribute 'point'
```

**Comment by saraedum** [comment]
With pplpy 0.8.7, these work fine.

**Comment by saraedum** [comment]
Seems to be fixed with SageMath 10.2. Sorry for the noise.

**Comment by saraedum** [comment]
Problem is not fixed yet in 10.1.
